marimo/_config/config.py

```diff
@@ -89,6 +89,18 @@ class DisplayConfig(TypedDict, total=False):
     theme: Literal["light", "dark"]
 
 
+class BrowserConfig(TypedDict, total=False):
+    """Settings related to web browsers.
+
+    **Keys.**
+
+    - `browser`: `"default"` or a browser registered with Python's
+        webbrowser module (e.g., `"firefox"` or `"google-chrome"`)
+    """
+
+    browser: Literal["default"] | str
```

Comment:
I wonder if this could go in runtime config? Or maybe we rename it to launch config?

Reply:
How about [server] instead? We'll likely add other server config, such as the port

Reply:
Yea I like that